color_refiment_helper.py

```diff
@@ -165,7 +165,15 @@ def get_unit_coloring(g: Graph) -> Coloring:
     return coloring
 
 
-def generate_neighbour_count_with_color(graph, current_color):
+def generate_neighbour_count_with_color(graph: Graph, current_color: "Integer") -> {}:
+    """
+    This methode creates a mapping from a vertex to the amount of neighbours with a color.
```

Comment:
`"Integer"` kan vervangen worden door `int`.

Ik zou `a color` vervangen door `current_color` en regel 171 kan dan weg.
Graag ook nog dingen invullen achter `:param`.
Bij result zou ik er een "mapping of colors to a vertex-neighbour_count mapping, the vertex-neighbour_count mapping is a dictionary itself which maps vertices to the amount of neighbours with current_color".